CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog][keepachangelog] and this project adheres to [Semantic Versioning][semver].
 
+## v4.0.0
+
+### Changed
+
+- Maximal `avtocod/b2b-api-php` package version now is `^4.0` (instead of `^3.3`)
```

Comment:
Maybe the `minimal` required version is `^4.0`?

In `composer.json` we limiting the minimum version, but not the maximum:
```
"avtocod/b2b-api-php": "^4.0",
```

Reply:
Yep